substrate/executor/src/wasm_executor.rs

```diff
@@ -473,7 +482,15 @@ impl CodeExecutor for WasmExecutor {
 		method: &str,
 		data: &[u8],
 	) -> Result<Vec<u8>> {
-		let module = Module::from_buffer(code).expect("all modules compiled with rustc are valid wasm code; qed");
+
+		let mut cache = MODULE_CACHE.lock().unwrap();
```

Comment:
codebase style with panickers is "prove or remove": either use `expect` with a string describing the assumptions that show why the panic won't happen, or don't potentially panic at all.

Either way it's irrelevant because we use `parking_lot` mutexes.